securedrop/journalist_app/__init__.py

```diff
@@ -18,15 +18,21 @@
 from models import Journalist
 from store import Storage
 
+MYPY = False
+if MYPY:
```

Comment:
A comment explaining the F401 exception is because of mypy would be better. if MYPY is too much of a hack.

Reply:
This is the formal way suggested by the mypy team for Python2.

Reply:
I will add the details on the same in the development docs, on how to add type annotation in our Python2 code.

Reply:
I don't understand why we have this `MYPY = False` here - can we add a comment explaining why this is the case? To the code or in the commit message